java/arcs/android/host/ArcHostHelper.kt

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.host
```

Comment:
Since this Class is intended to be used by ArcHost implementers, and not just arcs-internal usage, I think it would be better located in `arcs.sdk.android.host` instead.